rewriter/Visibility.cc

```diff
@@ -0,0 +1,44 @@
+#include "rewriter/Visibility.h"
+#include "ast/Helpers.h"
+#include "ast/ast.h"
+#include "core/Context.h"
+#include "core/Names.h"
+#include "core/core.h"
+#include "core/errors/rewriter.h"
+#include "rewriter/rewriter.h"
+
+using namespace std;
+
+namespace sorbet::rewriter {
+
+void Visibility::run(core::MutableContext ctx, ast::ClassDef *classDef) {
+    core::NameRef currentVisibility = core::Names::public_();
+
+    for (auto &stat : classDef->rhs) {
+        typecase(
+            stat.get(),
+            [&](ast::Send *send) {
+                if (send->args.size() != 0 && send->recv != nullptr) {
+                    return;
+                }
```

Comment:
It looks like this is currently prohibited:

```
class A
  self.private
  def foo; end
end
```

Reply:
By prohibited, do you mean that way of specifying orphaned calls will continue to not be respected by Sorbet?

Reply:
ah sorry i meant that my (now edited) test case will fail with this PR as written, but i think it should pass.